### PR TITLE
Add GET /ingest to ingest_tracker

### PR DESCRIPTION
**Pull request by kenoir**
Part of https://github.com/wellcomecollection/platform/issues/4419

### CODE REVIEW

File: ingests_tracker/src/main/scala/uk/ac/wellcome/platform/storage/ingests_tracker/IngestsTrackerApi.scala [hunk 1]
```diff
@@ -27,9 +35,19 @@ trait IngestsTrackerApi extends Runnable with Logging {
...
       get {
         pathPrefix("ingest" / JavaUUID) { id =>
-          // val result: ingestTracker.Result = ingestTracker.get(IngestID(id))
-
-          complete(StatusCodes.NotFound)
+          ingestTracker.get(IngestID(id)) match {
+            case Left(IngestDoesNotExistError(_)) =>
+              complete(StatusCodes.NotFound)
+            case Left(_) =>
+              complete(StatusCodes.InternalServerError)
```
Comment (alexwlchan) [inline ingests_tracker/src/main/scala/uk/ac/wellcome/platform/storage/ingests_tracker/IngestsTrackerApi.scala:42]: I can't remember – does this error get logged elsewhere?

File: ingests_tracker/src/test/scala/uk/ac/wellcome/platform/storage/ingests_tracker/IngestsTrackerApiFeatureTest.scala [hunk 2]
```diff
@@ -19,12 +23,54 @@ class IngestsTrackerApiFeatureTest
...
-        result.status shouldBe StatusCodes.NotFound
+          // We are interested in whether we can serialise/de-serialise
+          // what the JSON looks like is not interesting, so we
+          // do not need to test in detail as with the external API
+          withStringEntity(result.entity) { jsonString =>
+            fromJson[Ingest](jsonString).get shouldBe ingest
+          }
+        }
       }
     }
```
Comment (alexwlchan) [inline ingests_tracker/src/test/scala/uk/ac/wellcome/platform/storage/ingests_tracker/IngestsTrackerApiFeatureTest.scala:75]: Also needs a test for the HTTP 500 endpoint.

Comment (kenoir) [reply ingests_tracker/src/test/scala/uk/ac/wellcome/platform/storage/ingests_tracker/IngestsTrackerApiFeatureTest.scala:75]: 👍 is in next PR.

File: ingests_tracker/src/main/scala/uk/ac/wellcome/platform/storage/ingests_tracker/IngestsTrackerApi.scala (hunk 1 above)
Comment (kenoir) [reply ingests_tracker/src/main/scala/uk/ac/wellcome/platform/storage/ingests_tracker/IngestsTrackerApi.scala:42]: No, it should.

👍 is in next PR.

**Pull request merged**